packages/swapper/src/swappers/ChainflipSwapper/swapperApi/getTradeQuote.ts

```diff
@@ -130,7 +130,7 @@ export const getTradeQuote = async (
 
       const { data: swapResponse } = maybeSwapResponse.unwrap()
 
-      if (!swapResponse.id) throw Error('Missing Swap Id')
+      if (swapResponse.id === undefined) throw Error('Missing Swap Id')
```

Comment:
*not* gucci, can also be `null` which `!` guarded against but this won't
<img width="572" height="47" alt="image" src="https://github.com/user-attachments/assets/46d4315e-fc6a-455a-8054-7c877ab0dc75" />